common/configurable/src/main/java/io/helidon/common/configurable/spi/ExecutorServiceSupplierObserver.java

```diff
@@ -15,38 +15,35 @@
  */
 package io.helidon.common.configurable.spi;
 
-import java.lang.reflect.Method;
 import java.util.List;
 import java.util.concurrent.ExecutorService;
 import java.util.function.Supplier;
 
 /**
  * Behavior for observers of the various executor service suppliers.
  * <p>
- *     This component identifies suppliers to observers using:
- *     <ul>
- *         <li>the supplier itself,</li>
- *         <li>the supplier category (scheduled, server, ad-hoc), and</li>
- *         <li>the index of this supplier among suppliers in the same category.</li>
- *     </ul>
- *     Further, executor services furnished by the suppliers are identified to observers using:
- *     <ul>
- *         <li>the executor service itself, and</li>
- *         <li>the index of the executor service among those from the same supplier.</li>
- *     </ul>
- *     The consuming observers can use this identifying information however makes sense for them.
- *
+ * This component identifies suppliers to observers using:
```

Comment:
can we keep this in config/config instead of in common?

Reply:
I don't understand your comment. This class has been here before. It has nothing to do with config - it is support for observability of executors (for metrics).